build-constraints.yaml

```diff
@@ -899,7 +899,7 @@ packages:
         - grouped-list < 0 # GHC 8.4 via base-4.11.1.0
         - haskintex < 0
         - HaTeX < 0
-        - include-file
+        - include-file < 0 # https://github.com/Daniel-Diaz/include-file/pull/3
```

Comment:
Looks like this can be removed now, right? https://github.com/Daniel-Diaz/include-file/pull/3#issuecomment-461327401

Reply:
Sure, and there are some other updates